src/app/public/icons/icon-manifest.module.ts

```diff
@@ -2,13 +2,9 @@ import {
   NgModule
 } from '@angular/core';
 
-import {
-  SkyThemeIconManifestService
-} from './icon-manifest.service';
-
-@NgModule({
-  providers: [
-    SkyThemeIconManifestService
-  ]
-})
+/**
+ * @deprecated `SkyThemeIconManifestModule` is no longer needed
+ * and can be removed from your providers.
```

Comment:
"...and can be removed from your project." (They wouldn't be importing the module into their "providers" array?)